### PR TITLE
Copy SNMP profiles during installation

### PR DESCRIPTION
**Pull request by therve**
SNMP profiles are extra configuration files, we need to copy them during
installation.

### CODE REVIEW

File: omnibus/config/software/datadog-agent-integrations-py2.rb
Comment (olivielpeau) [inline omnibus/config/software/datadog-agent-integrations-py2.rb]: the `profile` var isn't defined (see errors in gitlab)

File: releasenotes/notes/snmp-profiles-9711c7d2307e9362.yaml
Comment (olivielpeau) [inline releasenotes/notes/snmp-profiles-9711c7d2307e9362.yaml]: I don't think this PR requires a mention in the changelog at all, since AFAIK it's only related to the integrations, so it's expected for a user who uses a version of an integration that includes SNMP profiles to see them in the Agent package that include that version of the integration

**Review by olivielpeau — COMMENTED** [review]
a typo, once fixed LGTM if the gitlab pipeline passes and the resulting packages have the files where you expect them

File: releasenotes/notes/snmp-profiles-9711c7d2307e9362.yaml
Comment (therve) [reply releasenotes/notes/snmp-profiles-9711c7d2307e9362.yaml]: Thanks, I changed it to `other`, is that enough?

File: omnibus/config/software/datadog-agent-integrations-py2.rb
Comment (therve) [reply omnibus/config/software/datadog-agent-integrations-py2.rb]: Fixed, thanks.

**Review by olivielpeau — DISMISSED** [review]
nits on the changelog, also please confirm the resulting packages in the pipeline have the files you expect in the right location.

Does the `datadog-agent integration` command need to be updated so that it copies these files as well?

**Review by olivielpeau — APPROVED** [review]
Approving, we talked on chat and you confirmed the package has the new files in the right locations 👍 

Please confirm whether the `integration` command needs an update as well before merging (if so, the update can be done in a separate PR)

**Review by FlorianVeaux — APPROVED** [review]
Looks good, the `integration` command will have to be updated as well

**Pull request merged**
